target/linux/realtek/dts-5.10/rtl838x.dtsi

```diff
@@ -152,6 +152,15 @@
 			#size-cells = <0>;
 		};
 
+		timer0: timer@3100 {
```

Comment:
any reason your not defining timer1-3? putting 'status = disabled' would be fine of course, but this is the main dtsi after all.

Reply:
We need only one timer for the whole system (see other implementations).